suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -58,13 +58,13 @@ tests:
       destroy-clster: false
       module: test_cephadm.py
       name: deploy cluster
- # Test cases to be executed
+# # Test cases to be executed
   - test:
       abort-on-fail: true
       config:
         command: add
         id: client.1
-        node: node4
+        node: node6
```

Comment:
why this been changed. conf file we are using is having only 4 nodes. There is no node6.